lib/services/ios-project-service.ts

```diff
@@ -221,11 +231,34 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				this.$logger.info("The iOS Deployment Target is now 8.0 in order to support Cocoa Touch Frameworks.");
 			}
 
-			let frameworkPath = this.getFrameworkRelativePath(libraryPath);
-			project.addFramework(frameworkPath, frameworkAddOptions);
+			let frameworkRelativePath = this.getLibSubpathRelativeToProjectPath(path.basename(frameworkPath));
+			project.addFramework(frameworkRelativePath, { customFramework: true, embed: true });
```

Comment:
You should use `frameworkAddOptions` here instead `{ customFramework: true, embed: true }`. There is some cases  when the framework is not dynamic and we we shouldn't embed it.